internal/indexer/indexer.go

```diff
@@ -18,9 +18,9 @@ type Indexer interface {
 // RegisterAll registers all known indexers to the given manager.
 // It uses the given logger to create a new named "indexer" logger,
 // passing that to each indexer.
-func RegisterAll(ctx context.Context, c cluster.Cluster, logger *zap.Logger) error {
+func RegisterAll(ctx context.Context, mgr manager.Manager, logger *zap.Logger) error {
```

Comment:
blocking: undo this change, we need to retain `c cluster.Cluster` as a parameter.

Reply:
Right! I missed this detail on the rebase, I will fix